Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkLoader.cpp

```diff
@@ -324,13 +326,13 @@ namespace AzToolsFramework
         }
     }
 
-    ViewBookmark ViewBookmarkLoader::GetLastKnownLocationInLevel() const
+    AZStd::optional<ViewBookmark> ViewBookmarkLoader::LoadLastKnownLocation() const
     {
-        return m_lastKnownLocation;
+        return AZStd::optional<ViewBookmark>(m_lastKnownLocation);
```

Comment:
update: It looks like `m_lastKnownLocation` itself should be an optional so we return that (and it will only hold a valid value when it's been written to)